packages/core/src/utils/index.ts

```diff
@@ -156,4 +160,24 @@ export function defineReadOnly<T, K extends keyof T>(
   });
 }
 
+/**
```

Comment:
I'd move this out to a diff file than index under utils so that index is just exports, but agree for taking it out of types

Reply:
good catch and +1, I started doing this with bigint.js because this file was getting crowded

Reply:
moved to userop.ts and export from index